Bootstrap Barrio (base theme) 5.5 broke AZParagraphsJavascriptTest

## Problem/Motivation
Looks like one of the PRs we merged into our temporary [bootstrap5 branch](https://github.com/az-digital/az_quickstart/tree/bootstrap5) introduced a legitimate test failure for our `AZParagraphsJavascriptTest`.

PR that introduced failure: #4306 (which upgraded Bootstrap Barrio to version 5.5)

We probably overlooked the failures because our `FunctionalJavascript` tests sometimes random failures on GitHub Actions runners.

All PRs targeting the `bootstrap5` branch are now seeing this same failure.

### Describe the bug
```
PHPUnit 9.6.23 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.3.22
Configuration: /home/runner/work/az_quickstart/az_quickstart/az-quickstart-scaffolding/web/profiles/custom/az_quickstart/phpunit.xml.dist

F                                                                   1 / 1 (100%)

Time: 04:21.613, Memory: 14.00 MB

There was 1 failure:

1) Drupal\Tests\az_paragraphs\FunctionalJavascript\AZParagraphsJavascriptTest::testParagraphs
TypeError: Cannot read properties of null (reading 'classList')
    at http://172.17.0.1:8080/themes/contrib/bootstrap_barrio/js/tabledrag.js?v=10.4.7:28:11
    at Array.forEach (<anonymous>)
    at Object.attach (http://172.17.0.1:8080/themes/contrib/bootstrap_barrio/js/tabledrag.js?v=10.4.7:25:45)
    at http://172.17.0.1:8080/core/misc/drupal.js?v=10.4.7:166:24
    at Array.forEach (<anonymous>)
    at Drupal.attachBehaviors (http://172.17.0.1:8080/core/misc/drupal.js?v=10.4.7:162:34)
    at success (http://172.17.0.1:8080/core/misc/ajax.js?v=10.4.7:1842:20)
    at l (http://172.17.0.1:8080/core/assets/vendor/loadjs/loadjs.min.js?v=4.3.0:1:217)
    at http://172.17.0.1:8080/core/assets/vendor/loadjs/loadjs.min.js?v=4.3.0:1:1712
    at Array.n (http://172.17.0.1:8080/core/assets/vendor/loadjs/loadjs.min.js?v=4.3.0:1:1639)
TypeError: Cannot read properties of null (reading 'classList')
    at http://172.17.0.1:8080/themes/contrib/bootstrap_barrio/js/tabledrag.js?v=10.4.7:28:11
    at Array.forEach (<anonymous>)
    at Object.attach (http://172.17.0.1:8080/themes/contrib/bootstrap_barrio/js/tabledrag.js?v=10.4.7:25:45)
    at http://172.17.0.1:8080/core/misc/drupal.js?v=10.4.7:166:24
    at Array.forEach (<anonymous>)
    at Drupal.attachBehaviors (http://172.17.0.1:8080/core/misc/drupal.js?v=10.4.7:162:34)
    at HTMLDivElement.<anonymous> (http://172.17.0.1:8080/core/misc/ajax.js?v=10.4.7:1402:18)
    at Function.each (http://172.17.0.1:8080/core/assets/vendor/jquery/jquery.min.js?v=3.7.1:2:3129)
    at ce.fn.init.each (http://172.17.0.1:8080/core/assets/vendor/jquery/jquery.min.js?v=3.7.1:2:1594)
    at Drupal.AjaxCommands.insert (http://172.17.0.1:8080/core/misc/ajax.js?v=10.4.7:1394:19)
TypeError: Cannot read properties of null (reading 'classList')
    at http://172.17.0.1:8080/themes/contrib/bootstrap_barrio/js/tabledrag.js?v=10.4.7:28:11
    at Array.forEach (<anonymous>)
    at Object.attach (http://172.17.0.1:8080/themes/contrib/bootstrap_barrio/js/tabledrag.js?v=10.4.7:25:45)
    at http://172.17.0.1:8080/core/misc/drupal.js?v=10.4.7:166:24
    at Array.forEach (<anonymous>)
    at Drupal.attachBehaviors (http://172.17.0.1:8080/core/misc/drupal.js?v=10.4.7:162:34)
    at http://172.17.0.1:8080/core/misc/ajax.js?v=10.4.7:1135:20

/home/runner/work/az_quickstart/az_quickstart/az-quickstart-scaffolding/web/core/tests/Drupal/FunctionalJavascriptTests/WebDriverTestBase.php:138
/home/runner/work/az_quickstart/az_quickstart/az-quickstart-scaffolding/vendor/phpunit/phpunit/src/Framework/TestResult.php:729

FAILURES!
Tests: 1, Assertions: 8, Failures: 1.
Error: Process completed with exit code 1.
```

## Proposed resolution
The test failure seems to be related to a change that was introduced in Bootstrap Barrio 5.5.x that extends the core tableDrag.js library:
https://www.drupal.org/project/bootstrap_barrio/issues/3294691

There also is an open issue on d.o. reporting errors that sound very similar to what's happening in our failing test.  The issue seems to be related to functionality provided by the Inline Entity form module.  There is also a patch available on the issue:
https://www.drupal.org/project/bootstrap_barrio/issues/3519060

We should probably try adding this patch for Bootstrap Barrio to see if it resolves the problem.

--- a/composer.json
+++ b/composer.json
@@ -146,6 +146,9 @@
             "drupal/auto_entitylabel": {
                 "Support entities without bundles (2945387)": "https://www.drupal.org/files/issues/2024-06-07/2945387-28.patch"
             },
+            "drupal/bootstrap_barrio": {
+                "Js error if table not found (3519060)": "https://www.drupal.org/files/issues/2025-04-14/bootstrap-barrio-table-not-found-3519060-1.patch"
+            },
             "drupal/cas": {
                 "ServiceNotFoundException when logging out (2948185)": "https://www.drupal.org/files/issues/cas_logout_error-2948185-2.patch"
             },

